Mathlib/Algebra/Category/ModuleCat/Sheaf/PushforwardContinuous.lean

```diff
@@ -40,4 +40,9 @@ noncomputable def pushforward : SheafOfModules.{v} R ⥤ SheafOfModules.{v} S wh
   map f :=
     { val :=(PresheafOfModules.pushforward φ.val).map f.val }
 
+/-- Given `M : SheafOfModules R` and `X : D`, this is the restriction of `M`
+over the sheaf of rings `R.over X` on the category `Over X`. -/
+noncomputable abbrev over (M : SheafOfModules.{v} R) (X : D) : SheafOfModules.{v} (R.over X) :=
+  (pushforward.{v} (𝟙 _)).obj M
+
```

Comment:
I know this isn't part of your PR, but line 30 of this file looks pretty weird to me. Is that really what was intended?

Reply:
You must be referring to the apparently duplicate assumption `[Functor.IsContinuous.{u} F J K] [Functor.IsContinuous.{v} F J K]`. Continuous functors are defined by saying that the "candidate" pushforward functor indeed sends sheaves of types to sheaves of types, but there is a universe parameter. I need both because for the degree of generality in this file, I need to pushforward both the sheaf of rings (in `Type u`) and the sheaf of modules (in `Type v`). In the case of schemes, we shall have `v = u`.